tool/e2e/pubspec.yaml

```diff
@@ -0,0 +1,14 @@
+name: e2e
```

Comment:
I'm not fully convinced about this directory strategy for e2e tests for `very_good_core`, but it doesn't disappoint me either. It was suggested by @renancaraujo .

Open to suggestions if anyone has better ideas.

Reply:
One other option is to have a `e2e` test directory where we can write our e2e tests and just have a specific github job running for it.

That would mean we have one package less to keep up to date as it would just be part of the current CLI package. But I can go either way

Reply:
I have no strong opinions here, shall we keep as it or do we want to do that change? (cc: @renancaraujo )